docs/podman-remote.1.md

```diff
@@ -0,0 +1,139 @@
+% podman-remote(1)
+
+## NAME
+podman - Simple management tool for pods, containers and images
+
+## SYNOPSIS
+**podman-remote** [*options*] *command*
+
+## DESCRIPTION
+Podman (Pod Manager) is a fully featured container engine that is a simple daemonless tool.
+Podman provides a Docker-CLI comparable command line that eases the transition from other
+container engines and allows the management of pods, containers and images.  Simply put: `alias docker=podman`.
+Most Podman commands can be run as a regular user, without requiring additional
+privileges.
+
+Podman uses Buildah(1) internally to create container images. Both tools share image
+(not container) storage, hence each can use or manipulate images (but not containers)
+created by the other.
+
```

Comment:
You talk all about Podman here, should probably have at least a paragraph or two on podman-remote and how it interacts with podman.

Reply:
Hmm okay I was talking to Dan about this, he mentioned that it should be the same as the regular podman documentation except with the commands that only applied to podman. This document also has a multi-use, since when it is put into the brew package, all the 'podman-remote's get changed to just podman; Dan said that this should be done like this since he does not want a podman-remote and a podman-remote-darwin as well. But this means that this document needs to cater to both a person using podman-remote and wanting to know the commands for podman-remote and a person using podman on a Mac (but really is just using podman-remote)

Reply:
Right on a MAC/Windows podman-remote does not exists, it is just podman.  We need to avoid talking about two different tools in the shared documents.

Reply:
Added a short paragraph about the remote working over varlink. When you say the commands, should all the commands in the chart be changed to podman-remote, ie ` [podman-remote-attach(1)](podman-attach.1.md)           | Attach to a running container.                                              |`

Reply:
No I think the top part is fine.  I would leave the chart alone.